dspace-api/src/main/java/org/dspace/content/virtual/VirtualMetadataConfiguration.java

```diff
@@ -42,4 +42,8 @@
      * @return  The useForPlace to be used by this bean
      */
     boolean getUseForPlace();
+
+    void setPopulateWithNameVariant(boolean populateWithNameVariant);
```

Comment:
Could you please add missing java docs?

Reply:
This is not a public method, but JavaDocs have been included now

Reply:
The JavaDocs have been included